src/applications/edu-benefits/0993/containers/SinglePageForm.jsx

```diff
@@ -0,0 +1,183 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+import Scroll from 'react-scroll';
+import _ from 'lodash/fp';
+import classNames from 'classnames';
+
+import SubmitController from './SubmitController';
```

Comment:
This component is essentially the same as FormPage (the default form page component) except that it uses a custom SubmitController rather than the standard ProgressButtons, to support submission on the first and only form page.